include/io/mman-win32/mman.h

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Could you add where these files are from? Perhaps http://mman-win32.googlecode.com/svn/trunk/?